README.md

```diff
@@ -208,6 +216,14 @@ Generated by [@ucanto/principal `EdSigner`](https://github.com/web3-storage/ucan
 
 _Example:_ `MgCZG7EvaA...1pX9as=`
 
+#### `CONTENT_CLAIMS_PRIVATE_KEY`
+
+The [`multibase`](https://github.com/multiformats/multibase) encoded ED25519 keypair used as the signing key for the [content-claims api](https://github.com/web3-storage/content-claims).
+
+Generated by [@ucanto/principal `EdSigner`](https://github.com/web3-storage/ucanto) via [`ucan-key`](https://www.npmjs.com/package/ucan-key)
+
+_Example:_ `MgCZG7EvaA...1pX9as=`
+
```

Comment:
One thing I did when integrating claims with the current web3.storage API was allow a proof to be passed. 

You need to sign a UCAN with a private key _and_ provide proof that you have the capability. When the private key is the private key of the content claims service no proof is needed (you are self signing). When it's a different private key you need to have a delagation for `assert/equals` as proof.

By allowing a proof to be passed you enable both cases. It just means we can switch to the latter case in the future without code changes.

Not blocking...but should be easy to implement:

https://github.com/web3-storage/web3.storage/blob/1cc9707af6a10bccec4dd93888cf1d5fc1a8c0cb/packages/api/src/env.js#L232-L248